### PR TITLE
feat(feature added): more functions added in the package

### LINKED ISSUE #10: List all labels
Create method `getLabels()`

It must work for the following endpoint:
- https://en.wiki.openfoodfacts.org/API#List_of_labels

### PR DESCRIPTION
**Pull request by ms10398**
Functions added in the commit include
- add getLabels() function
- add getProductByBarcodeBeginning(begin) function
- add getLanguages() function
- add getBrand(brandName) function

Fixes #10 #9 #6 #5 

Ping @kozlown 

Have a look at it

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -14,15 +14,49 @@ class OFF {
...
+      .then(JSON.parse)
+  }
+
+  getProductsByBarcodeBeginning (begin) {
+    const URL = this.URL
+    this.URL = 'https://world.openfoodfacts.org'
+    let noOfX = 13 - begin.length
+    let barcode = begin
+    for (let x = 1; x <= noOfX; x++) { barcode += 'x' }
+    console.log(barcode)
```
Comment (kozlown) [inline main.js]: you can remove this console.log

File: main.js [hunk 2]
```diff
@@ -14,15 +14,49 @@ class OFF {
...
+    return request(`${URL}/labels.json`)
+      .then(JSON.parse)
+  }
+
+  getProductsByBarcodeBeginning (begin) {
+    const URL = this.URL
+    this.URL = 'https://world.openfoodfacts.org'
+    let noOfX = 13 - begin.length
+    let barcode = begin
+    for (let x = 1; x <= noOfX; x++) { barcode += 'x' }
```
Comment (kozlown) [inline main.js]: lines 53-55 can be resumed to :

```js
const fill = 'x'.repeat(13 - begin.length)
const barcode = begin.concat(fill)
```

File: main.js [hunk 3]
```diff
@@ -14,15 +14,49 @@ class OFF {
...
+  }
+
+  getLabels () {
+    const URL = this.URL
+    this.URL = 'https://world.openfoodfacts.org'
+    return request(`${URL}/labels.json`)
+      .then(JSON.parse)
+  }
+
+  getProductsByBarcodeBeginning (begin) {
```
Comment (kozlown) [inline main.js]: Rename `begin` (verb) to `beginning` (noun)

File: main.js [hunk 4]
```diff
@@ -14,15 +14,49 @@ class OFF {
```
Comment (kozlown) [inline main.js]: You can remove

```js
const URL = this.URL
this.URL = 'https://world.openfoodfacts.org'
```
from all methods,`this.URL` is defined in the `constructor`, you only have to use it like : 
```js
 return request(`${this.URL}/brands.json`)
      .then(JSON.parse)
```

Comment (ms10398) [reply main.js]: @kozlown As we are making one object and using its URL property everywhere so if we alter the URL using country function and chain then URL is altered thats why I reset the URL after every function.

Do you have any special trick in hand for that.

Comment (ms10398) [reply main.js]: I was thinking either we can make a reset function or whats your thought

File: main.js (hunk 3 above)
Comment (ms10398) [reply main.js]: Yes I would be doing it

File: main.js (hunk 2 above)
Comment (ms10398) [reply main.js]: Will try to implement it like this

File: main.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ms10398) [reply main.js]: Yes sure :sparkles:

File: main.js (hunk 4 above)
Comment (kozlown) [reply main.js]: Delayed to #13

**Pull request merged**